analyzers/src/SonarAnalyzer.CSharp/Rules/ConditionalStructureSameImplementation.cs

```diff
@@ -32,11 +32,11 @@ namespace SonarAnalyzer.Rules.CSharp
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public sealed class ConditionalStructureSameImplementation : ConditionalStructureSameImplementationBase
     {
-        private static readonly DiagnosticDescriptor rule =
```

Comment:
If you are already doing a cleanup of the classes, I think you could move the `DiagnosticDescriptor` and the `SupportedDiagnostics` to the base class, like [here](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.Common/Rules/ConditionalStructureSameConditionBase.cs)